### DISCUSSION TITLE
How to create conditional questionaires

### DISCUSSION THREAD

**Discussion by TjebbeVQ** [globaleaks/globaleaks-whistleblowing-software#3430] [Q&A]
How would we go about creating dynamic questionnaires based on triggers/previously answered questions?

Thanks!

**Comment by evilaliv3** [comment]
To create a conditional questions you need two questions:
- a question to act as a trigger of type selectbox or checkbox
- a question to be triggered when a specific options of the trigger is selected; on this question you should add a trigger configuration.

**Comment by robertodc** [comment]
Hello! I'm trying to create in teh demo program a question with 2 different answeres, yes or not.
If yes is chosen, then the program goes forward, or opens other questions.
If not is chosen, then the program gives alert, saying: you must read and accept, otherwise close the program.
How is it possible?
thank you.
